Src/CSharpier.Tests/FormattingTests/TestFiles/cs/UsingDirectives.test

```diff
@@ -1,29 +1,13 @@
 // leading
-using First; // trailing
-
+using A; // trailing
 // leading with space
 using // another trailing
-Second;
-
-using
-// static leading
-static // static trailing
-Third;
-
-using M = System.Math;
-using Point = (int x, int y);
-
-using static System.Math;
-
-global using System;
-
-using First;
-using Second;
+B;
```

Comment:
wtf is this?

Reply:
I believe that was me going nuts trying comments everywhere a while back.
Apparently that is valid c#!